PSICover/PSICover.cs

```diff
@@ -163,31 +166,91 @@ class Analyzer {
          var code = File.ReadAllLines (file);
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
+         int hitCount = 0;
          foreach (var block in blocks) {
             bool hit = hits[block.Id] > 0;
-            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\">";
+            if (hit) hitCount++;
+            string hitStyles = hit ? "hit tooltip" : "unhit";
+            string tag = $"<span class=\"{hitStyles}\">";
+            // Another solution, But this does not provide indentation.
+            //if (block.ELine - block.SLine > 0) tag = tag.Replace (">", " style=\"white-space:pre-line\">");
+
+            // Break multiple lines and attach span element to each line.
+            var linesCount = block.ELine - block.SLine;
+            if (linesCount > 0) {

```

Comment:
Special case for multi line is not required. You could use a single for loop and if condition inside for loop to achieve the result.